### PR TITLE
Add and fix promise information

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Staged links
[Migration Overview](https://deploy-preview-3845--developer-chrome-com.netlify.app/docs/extensions/mv3/intro/mv3-overview#promises)
[Promises](https://deploy-preview-3845--developer-chrome-com.netlify.app/docs/extensions/mv3/promises)

### CODE REVIEW

File: site/en/docs/extensions/mv3/intro/mv3-overview/index.md [hunk 1]
```diff
@@ -130,7 +130,8 @@ to an API method, this prevents the promise from being returned. Therefore you
 can defer this part of your migration until you're ready, or begin using
 promises immediately.
 
-Some scenarios, such as event listeners, will still require callbacks.
+Some scenarios, such as event listeners, will still require callbacks. For
+more information, see [Promises on MDN][3].
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/intro/mv3-overview/index.md:134]: MDN's article on promises is a great addition! 👍

I'm wondering if we should also add a link to the [Using promises](https://deploy-preview-3845--developer-chrome-com.netlify.app/docs/extensions/mv3/promises#compare-to-callback) article that explains how to convert from callbacks to promises 🤔. Just a thought

**Review by AmySteam — APPROVED** [review]
Thanks! LGTM 👍 

Just had one suggestion, but not significant enough to block this PR.

File: site/en/docs/extensions/mv3/intro/mv3-overview/index.md (hunk 1 above)
Comment (jpmedley) [reply site/en/docs/extensions/mv3/intro/mv3-overview/index.md]: Done.

File: site/en/docs/extensions/mv3/intro/mv3-overview/index.md [hunk 2]
```diff
@@ -130,7 +130,9 @@ to an API method, this prevents the promise from being returned. Therefore you
 can defer this part of your migration until you're ready, or begin using
 promises immediately.
 
-Some scenarios, such as event listeners, will still require callbacks.
+Some scenarios, such as event listeners, will still require callbacks. For
+information on using promises, see [Promises on MDN][3]. For information on 
+converting callbacks to promises, see [our ouwn article][4].
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/intro/mv3-overview/index.md:135]:
```suggestion
converting callbacks to promises, see [our own article][4].
```

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**